homeassistant/components/sonos/alarms.py

```diff
@@ -0,0 +1,70 @@
+"""Class representing Sonos alarms."""
+from __future__ import annotations
+
+from collections.abc import Iterator
+import logging
+
+from pysonos import SoCo
+from pysonos.alarms import Alarm, get_alarms
+from pysonos.exceptions import SoCoException
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+from .const import DATA_SONOS, SONOS_ALARMS_UPDATED, SONOS_CREATE_ALARM
+from .household_coordinator import SonosHouseholdCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SonosAlarms(SonosHouseholdCoordinator):
+    """Storage class for Sonos alarms."""
+
+    def __init__(self, hass: HomeAssistant, household_id: str) -> None:
```

Comment:
How about using `**kwargs` here? This way we would leave the arguments to the parent?